src/Defs.h

```diff
@@ -0,0 +1,8 @@
+#pragma once
```

Comment:
this header contains some misc around the prj,
would rather to split to different (with meaningful naming) header or namespaces